src/main/resources/js/lgtm-addon.js

```diff
@@ -78,69 +79,68 @@ function loadProjects() {
 }
 
 function handleProjectChange(event) {
-	clearSelect2Field(issueTypeFieldId);
+	lgtmIssueTypeId = null;
 	clearSelect2Field(priorityFieldId);
 	clearSelect2Field(closedStatusFieldId);
 	clearSelect2Field(reopenedStatusFieldId);
 	
 	if (AJS.$(projectFieldId).select2('val') !== "none") {
-		loadIssueTypes(AJS.$(projectFieldId).select2('val'));
+		loadIssueType(AJS.$(projectFieldId).select2('val'));
 		loadPriorities(AJS.$(projectFieldId).select2('val'));
 	}
 }
 
-function loadIssueTypes(projectKey) {
+function loadIssueType(projectKey) {
 	$.ajax({
 		url : AJS.contextPath() + "/rest/api/2/project/" + projectKey + "/statuses"
 	}).done(
 	function(issueTypes) {
-		var fieldContent = issueTypes.map(function(issueType) {
-			return {
-				value : issueType.id,
-				text : issueType.name
-			}
+		var lgtmIssueType = issueTypes.find(function (element) {
+			return element.name === "LGTM Alert";
 		});
-		fieldContent.unshift({value : "none", text : ""});
 		
-		renderSelect2Field(issueTypeFieldId, fieldContent);
-		AJS.$(issueTypeFieldId).on('change', function(e) {
-			handleIssueTypeChange(issueTypes);
-		});
-	
-		if (config !== null) {
-			changeSelect2Value(issueTypeFieldId, config.issueTypeId);
+		if (lgtmIssueType === undefined) {
+			AJS.messages.error("#message-context", {
+				title : 'The project ' + projectKey + ' uses an issue type scheme that does not contain the "LGTM alert" issue type.', 
```

Comment:
Something odd happens with the alert box when you select a project twice...
![image](https://user-images.githubusercontent.com/1482231/52483460-e4d3d280-2bab-11e9-9b4a-d966a7d90182.png)

Reply:
It also doesn't clear on reselecting the project after having followed the instructions. I guess people might expect to have to refresh, but is there a way for selecting a project to clear any box that's already present?